### PR TITLE
updated endpoints

### PR DESCRIPTION
**Pull request by RyanSWest**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: quake/src/components/Login.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Login = () => {
     e.preventDefault();
     // axiosWithAuth()
     axios
-      .post("http://localhost:3300/api/users/login", user)
+      .post("http://epicentralpt9.herokuapp.com/api/users/login", user)
```
Comment (RyanSWest) [inline quake/src/components/Login.js:18]: ddd

File: quake/src/components/Login.js [hunk 2]
```diff
@@ -50,9 +50,10 @@ const Login = () => {
           onChange={handleInput}
           placeholder="password"
         />
-        <button className = 'enter' type="submit">Enter</button>
+        <button data-testid="EnterButton"
```
Comment (JJAleman) [inline quake/src/components/Login.js:53]: testing button

File: quake/src/components/Login.js [hunk 3]
```diff
@@ -50,9 +50,10 @@ const Login = () => {
...
+ className = 'enter' id = 'button' type="submit">Enter</button>
```
Comment (JJAleman) [inline quake/src/components/Login.js:54]: testing button

File: quake/src/components/Login.js [hunk 4]
```diff
@@ -50,9 +50,10 @@ const Login = () => {
...
         <h3>Not Registered?</h3>
-        <NavLink className = 'small-nav' to = "/register">Register</NavLink>
+        {/* <NavLink className = 'small-nav' to = "/register">Register</NavLink> */}
```
Comment (JJAleman) [inline quake/src/components/Login.js:56]: Commented out Register link

**Comment by JJAleman** [comment]
The file structure was from an old version.

**Pull request closed without merging**